parent/pom.xml

```diff
@@ -1237,6 +1237,40 @@
                 <artifactId>elasticsearch-maven-plugin</artifactId>
                 <version>${version.elasticsearch.plugin}</version>
              </plugin>
+             <plugin>
+                <groupId>org.apache.karaf.tooling</groupId>
+                <artifactId>karaf-maven-plugin</artifactId>
+                <version>${version.karaf-maven-plugin}</version>
+                <configuration>
+                   <descriptors>
+                      <descriptor>file://${project.build.directory}/classes/features.xml</descriptor>
+                      <!-- ideally, those profiles should be referenced in the features file -->
```

Comment:
yes. the most portable way to define a feature xml node is to include in each of them a reference to the features definition of any dependency they might refer to. This would make them entirely self contained, at the price of some verbosity and the effort of keeping them in sync, if you have multiple features.xml files.
This is a different approach to define these dependencies in a centralized place.